### DISCUSSION TITLE
app of app of apps?

### DISCUSSION THREAD

**Discussion by ArjunDandagi** [argoproj/argo-cd#8038] [Q&A]
i Would like to create a root of of root app which inturn will have other apps. 
why? we kinda deploy multiple instance of same object in the same cluster with suffix like x, y, z
which will be nginx-x , nginx-y, nginx-z 
and i dont want to have them as part of the root app separate since they all simply use the same image ( x,y,z are neighbour countries) . so can i create a app NGINX . that will have these three of that? 
does this complicate the managing of resources 
this is the structure i want 
```
---> root.yml 
              ----> nginx.yaml 
                             ----> nginx-x.yml 
                             ----> nginx-y.yml
                             ----> nginx-z.yml 
                            ----> deploys/nginx-pod.yaml 
              ------> apache.yaml 
              ------> anotherapp.yaml
```

**Comment by RomanBats** [comment]
I don't think that I got this question right, but, You [cannot share a resource](https://github.com/argoproj/argo-cd/issues/2600) with several ArgoCD applications.
What I think should work for you is to have root-app that targets a helm chart, that contains apps, one of which is an app that targets a helm chart :D :
![image](https://user-images.githubusercontent.com/5024978/147913738-8dc22e8e-d238-439a-9be1-71863b332452.png)
